material-ui-chip-input 1.0.0-beta.9 is breaking webpack compilation

**What you were expecting:**
Compiling files with webpack to produce a successful build

**What happened instead:**
I get this error 
```bash
Failed to compile.

./node_modules/material-ui-chip-input/lib/ChipInput.js
Module not found: Can't resolve '@material-ui/core/FilledInput/FilledInput' in '<path>/node_modules/material-ui-chip-input/lib'
```

**Other information:**
This appears related to the latest beta release of [material-ui-chip-input](https://github.com/TeamWertarbyte/material-ui-chip-input/commit/fbf8499c4f3f6f4f5c6fd0ca324778dc00f7c40a) which adds FilledInput and OutlinedInput which are not present in @material-ui/core 1.x

**Environment**

* React-admin version: 2.4.3
* React version: 16.6.3

I think the quick fix would be to pin material-ui-chip-input to 1.0.0-beta.8 or else update @material-ui/core to 3.x

--- a/packages/ra-ui-materialui/package.json
+++ b/packages/ra-ui-materialui/package.json
@@ -47,7 +47,7 @@
         "classnames": "~2.2.5",
         "inflection": "~1.12.0",
         "lodash": "~4.17.5",
-        "material-ui-chip-input": "^1.0.0-beta.6",
+        "material-ui-chip-input": "1.0.0-beta.6 - 1.0.0-beta.8",
         "papaparse": "^4.1.4",
         "prop-types": "~15.6.1",
         "ra-core": "^2.4.3",

